examples/DDPG/ou_strategy.py

```diff
@@ -0,0 +1,44 @@
+#   Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Reference: https://github.com/rll/rllab/blob/master/rllab/exploration_strategies/ou_strategy.py
+"""
+
+import numpy as np
+
+
+class OUStrategy(object):
```

Comment:
Could you explain why not use white noise for exploration? 
Is this exploration better than a zero mean, independent covariance gaussian noise?
and why?

Reply:
In continuous control task, it may be better to have temporally correlated exploration. I will make up experiments to compare the two exploration ways.

Reply:
In the HalfCheetah-v2 game, there is no big difference in the performance between ou noise and white noise, so will use the white noise which is simpler.